sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/collect.scala

```diff
@@ -86,7 +86,13 @@ abstract class Collect extends ImperativeAggregate {
  * Collect a list of elements.
  */
 @ExpressionDescription(
-  usage = "_FUNC_(expr) - Collects and returns a list of non-unique elements.")
+  usage =
+    """
+      _FUNC_(expr) - Collects and returns a set of unique elements.
```

Comment:
Should this say 'list'? and it doesn't seem like they're unique, necessarily, given existing text?